mclf/berkovich/berkovich_line.py

```diff
@@ -1251,7 +1248,7 @@ def point_in_between(self, xi1):
             s2 = s0 + 1
         else:
             s2 = (s0+s1)/2
-        xi2 = X.point_from_discoid(phi, s2, in_unit_disk)
+        xi2 = xi1._X.point_from_discoid(phi, s2, in_unit_disk)
```

Comment:
@swewers What is supposed to happen here?

Reply:
Yes, `X` should be replaced by `xi1._X` (or `xi0._X`)
It is strange that this has never caused an error. The function `point_in_between` is used only one time, by `union` in `affinoid_domains`, but it seems that the case where it is used almost never occurs. 
Clearly there need to be more doctests!